### PR TITLE
Create bruce.py

### PR DESCRIPTION
**Pull request by lennartwesdijk**

### CODE REVIEW

File: plugins/bruce.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import requests
+
+from will.plugin import WillPlugin
+from will.decorators import respond_to
+
+class CryptoPlugin(WillPlugin):
+    @respond_to("crypto")
```
Comment (joshp123) [inline plugins/bruce.py:7]: change it to 

```
 @hear("^pan$)"
```

Comment (joshp123) [inline plugins/bruce.py:7]: well crpyto instead of pan

Comment (joshp123) [inline plugins/bruce.py:7]: this matches just the word by itself on a line as opposed to matching on "cryptohackers" or "scyrpto"

Comment (lennartwesdijk) [inline plugins/bruce.py:7]: Didn't want to make this obtrusive, so just @byndiebot crypto

Comment (joshp123) [inline plugins/bruce.py:7]: oh sorry it's respond to not hear

Comment (joshp123) [inline plugins/bruce.py:7]: my bad

**Pull request merged**